common/models/acl.js

```diff
@@ -376,6 +376,15 @@ module.exports = function(ACL) {
   };
 
   /**
+   * Test if ACL permission is not DENY
+   * @returns {Boolean} is permission type different than 'DENY' ?
+   */
+
+  ACL.prototype.isAllowed = function() {
+    return this.permission !== loopback.ACL.DENY;
```

Comment:
> I would expect that such request will be allowed, and my expectation as a LoopBack user would be that role2 is not included in authorizedRoles

agreed: this is the result one would expect. I'll check my code to make sure it is always the case and work on test cases when possible